bazel/dependencies.bzl

```diff
@@ -36,9 +36,9 @@ def proxy_wasm_cpp_host_dependencies():
         extra_target_triples = [
             "aarch64-unknown-linux-gnu",
             "wasm32-unknown-unknown",
-            "wasm32-wasi",
+            "wasm32-wasi", # TODO: Change to wasm32-wasip2 once https://github.com/bazelbuild/rules_rust/issues/2782 is fixed
```

Comment:
We're using a small subset of hostcalls from WASI preview1 in Proxy-Wasm.

Why are you suggesting using preview2 here? Is that a Wasmtime dependency? The WASI support in Wasmtime should be disabled, otherwise it might conflict with our implementations, or we might accidentally expose hostcalls that shouldn't be exposed in Proxy-Wasm environments.

Reply:
Whoops I meant wasm32-wasip1 since that's the alias. Good catch!